client/globals.d.ts

```diff
@@ -89,3 +90,5 @@ declare global {
 
 	const wcSettings: Record< string, any >;
 }
+
+declare const wcTracks: any;
```

Comment:
Let's see why this was added as I didn't have it in my PR.

Reply:
Removed on [815bd03](https://github.com/Automattic/woocommerce-payments/pull/6275/commits/815bd03f1153b5d01fa6c03f78edd9b47fac1fc7). It was probably a leftover from a merge conflict resolution. Thanks for catching it!